### PR TITLE
change search and notification bell icon size

### PR DESCRIPTION
**Pull request by snehalsabade**

### CODE REVIEW

File: packages/nulp_elite/src/App.js [hunk 1]
```diff
@@ -14,10 +14,9 @@ import { DEFAULT_THEME, initializeI18n,H2, } from "@shiksha/common-lib";
 import { useTranslation } from 'react-i18next'
 import Framework from "pages/Frmework";
 import User from "pages/User";
-import { extendTheme, NativeBaseProvider } from "native-base";
-// import dotenv from "dotenv";
-import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+import { extendTheme } from "native-base";
```
Comment (paritshivani) [inline packages/nulp_elite/src/App.js:17]: Why it is needed? @snehalsabade

File: packages/nulp_elite/src/pages/Sample.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ import { useTranslation } from "react-i18next";
 import moment from "moment";
 import manifest from "../../src/manifest.json";
 import "ag-grid-community/styles/ag-grid.css";
-import { ChakraProvider, CSSReset, Box, Container, Button } from '@chakra-ui/react';
+import { ChakraProvider, CSSReset, Container } from '@chakra-ui/react';
```
Comment (paritshivani) [inline packages/nulp_elite/src/pages/Sample.js:19]: Try to remove chakra UI import if we are not using it

File: yarn.lock
Comment (paritshivani) [inline yarn.lock:1]: Dont comment yarn.lock file until you have installed any new dependancy

File: packages/nulp_elite/src/App.js (hunk 1 above)
Comment (snehalsabade) [reply packages/nulp_elite/src/App.js:17]: We are not using extendTheme so I can remove it.

File: packages/nulp_elite/src/pages/Sample.js (hunk 2 above)
Comment (snehalsabade) [reply packages/nulp_elite/src/pages/Sample.js:19]: Removed

File: yarn.lock
Comment (snehalsabade) [reply yarn.lock:1]: Sure

**Pull request merged**